privateSBOMExchange/src/petra/lib/internals/producer.py

```diff
@@ -42,10 +42,10 @@ def get_producer_keys(self):
         resp = requests.post(f"{self.kms_url}/provision-producer-keys")
         if resp.status_code != 200:
             raise Exception(f"Failed to provision key: {resp.text}")
-        cpabe_sk, signing_key, cert = resp.json().get("cpabe_sk"), resp.json().get("signing_key"), resp.json().get("cert")
-        if not all([cpabe_sk, signing_key, cert]):
-            raise Exception("Failed to get cpabe_sk, signing key or certificate from KMS")
-        self.cpabe_sk, self.signing_key, self.signing_cert = cpabe_sk, signing_key, cert
+        cpabe_sk, signing_key, cert,epoch_info= resp.json().get("cpabe_sk"), resp.json().get("signing_key"), resp.json().get("cert"),resp.json().get("epoch_info")
+        if not all([cpabe_sk, signing_key, cert,epoch_info]):
```

Comment:
Styling nit: For code readability, we should be consistent about adding a space between variables and values